src/main/java/com/impossibl/postgres/system/BasicContext.java

```diff
@@ -288,7 +288,7 @@ private void loadLocale() throws IOException, NoticeException {
 
       // Check if locale matches a win32 type locale (e.g. "English_United States.1252")
       if (Locales.getJavaCompatibleLocale(localeSpec.split("\\.")[0]) != null) {
-        localeSpec = Locales.getJavaCompatibleLocale(localeSpec);
+        localeSpec = Locales.getJavaCompatibleLocale(localeSpec.split("\\.")[0]);
```

Comment:
Hi,

Maybe extract that to a variable to avoid doing the operation twice (and making the same mistake again in the future)?